### PR TITLE
[TGL-H] PV Upstream

### PR DESCRIPTION
**Pull request by japelner-intc**
This patch adds support for TGL-H in both Platform and Silicon packages.

Signed-off-by: Jim <jim.pelner@intel.com>

### CODE REVIEW

File: Platform/TigerlakeBoardPkg/Script/StitchIfwiConfig_tglh.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  3. Openssl
+       Openssl is required for stitch. the stitch tool will search evn OPENSSL_PATH,
+       to find Openssl. If evn OPENSSL_PATH is not found, will find openssl from
+       "C:\\Openssl\\Openssl"
+
+  4. Stitch the final image
+       EX:
+       Assuming stitching workspace is at D:\Stitch and building ifwi for TGL platform
+       To stitch IFWI with SPI QUAD mode and Boot Guard profile VM:
+         StitchIfwi.py -b vm -p tglu -w D:\Stitch -s Stitch_Components.zip -c StitchIfwiConfig.py
```
Comment (gdong1) [inline Platform/TigerlakeBoardPkg/Script/StitchIfwiConfig_tglh.py:40]: the tglh example use "tglh" instead of "tglu"

Comment (japelner-intc) [reply Platform/TigerlakeBoardPkg/Script/StitchIfwiConfig_tglh.py:40]: Thank for the catch, will get that fixed in the next patch.

File: Platform/TigerlakeBoardPkg/BoardConfig.py [hunk 2]
```diff
@@ -202,7 +202,15 @@ def __init__(self, *args, **kwargs):
...
-        self._CFGDATA_EXT_FILE = [self._generated_cfg_file_prefix + 'CfgData_Int_Tglu_Ddr4.dlt', self._generated_cfg_file_prefix  + 'CfgData_Int_Tglu_DdrLp4.dlt']
+        self._CFGDATA_EXT_FILE = [self._generated_cfg_file_prefix + 'CfgData_Int_Tglu_Ddr4.dlt', self._generated_cfg_file_prefix  + 'CfgData_Int_Tglu_DdrLp4.dlt', self._generated_cfg_file_prefix  + 'CfgData_Int_Tglh_Ddr4.dlt']
+
+        # If mulitple VBT table support is required, list them as:
+        #   {VbtImageId1 : VbtFileName1, VbtImageId2 : VbtFileName2, ...}
+        # VbtImageId is ID to identify a VBT image. It is a UINT32 number to match
+        #   the ImageId field in the VBT container.
+        # VbtFileName is the VBT file name. It needs to be located under platform
+        #   VbtBin folder.
+        self._MULTI_VBT_FILE      = {1:'Vbt.dat', 2:'Vbt_tglH_Hdmi.bin'}
```
Comment (gdong1) [inline Platform/TigerlakeBoardPkg/BoardConfig.py:213]: Often we use "dat" file extension for VBT file to indicate it is a data config file, not an executable binary file.
To keep consistence, had better rename it to "Vbt_tglH_Hdmi.dat"

Comment (japelner-intc) [reply Platform/TigerlakeBoardPkg/BoardConfig.py:213]: Got it and agree. Thanks.

**Review by gdong1 — APPROVED** [review]

**Pull request merged**